### PR TITLE
contextId -> iTwinId as per core

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

File: packages/modules/cra-template-web-viewer/template/src/App.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const App: React.FC = () => {
       false
   );
   const [oidcInitialized, setOidcInitialized] = useState(false);
-  const [contextId, setContextId] = useState(process.env.IMJS_CONTEXT_ID);
+  const [iTwinId, setContextId] = useState(process.env.IMJS_CONTEXT_ID);
```
Comment (aruniverse) [inline packages/modules/cra-template-web-viewer/template/src/App.tsx:44]: should the env var be renamed too? i think so but just wanted to confirm

File: packages/apps/web-viewer-test/src/components/home/AuthConfigHome.tsx [hunk 2]
```diff
@@ -26,12 +26,12 @@ export const AuthConfigHome: React.FC = () => {
...
     (BaseInitializer.authClient?.hasSignedIn &&
       BaseInitializer.authClient?.isAuthorized) ||
-    false
+      false
   );
   const [iModelId, setIModelId] = useState(
     process.env.IMJS_AUTH_CLIENT_IMODEL_ID
   );
-  const [contextId, setContextId] = useState(
+  const [iTwinId, setContextId] = useState(
```
Comment (kckst8) [inline packages/apps/web-viewer-test/src/components/home/AuthConfigHome.tsx:34]:
```suggestion
  const [iTwinId, setITwinId] = useState(
```

File: packages/apps/web-viewer-test/src/components/home/IModelBankHome.tsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ export const IModelBankHome: React.FC = () => {
   const [iModelId, setIModelId] = useState(
     process.env.IMJS_AUTH_CLIENT_IMODEL_ID
   );
-  const [contextId, setContextId] = useState(
+  const [iTwinId, setContextId] = useState(
```
Comment (kckst8) [inline packages/apps/web-viewer-test/src/components/home/IModelBankHome.tsx:39]:
```suggestion
  const [iTwinId, setITwinId] = useState(
```

File: packages/modules/cra-template-web-viewer/template/src/App.tsx (hunk 1 above)
Comment (kckst8) [reply packages/modules/cra-template-web-viewer/template/src/App.tsx:44]: yes. Might as well

File: packages/modules/web-viewer-react/README.md [hunk 4]
```diff
@@ -65,13 +65,13 @@ export const MyViewerComponent = () => {
 
 #### Required
 
-- `contextId` - GUID for the context (project, asset, etc.) that contains the iModel that you wish to view
+- `iTwinId` - GUID for the context (project, asset, etc.) that contains the iModel that you wish to view
```
Comment (kckst8) [inline packages/modules/web-viewer-react/README.md:68]:
```suggestion
- `iTwinId` - GUID for the iTwin (project, asset, etc.) that contains the iModel that you wish to view
```

**Review by kckst8 — APPROVED** [review]
Few suggestions to address prior to merging. Otherwise looks great

**Review by kckst8 — CHANGES_REQUESTED** [review]
Would need to update uses of setContextId where changed

**Review by kckst8 — APPROVED** [review]

**Pull request merged**